scriptmodules/supplementary/runcommand/joy2key.py

```diff
@@ -114,8 +187,10 @@ def process_event(event):
 
 signal.signal(signal.SIGINT, signal_handler)
 
+js_button_codes = {}
```

Comment:
This dictionary will be used to store button codes for each connected controller using. The dictionary keys will be the file descriptors obtained in `open_devices()`.